custom_fsync/sync.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"fmt"
+	"os"
+)
+
+func main() {
+	if len(os.Args) != 1 {
+		fmt.Fprintf(os.Stderr, "Usage: %s\n", os.Args[0])
+		os.Exit(1)
+	}
+
+	files, err := os.ReadDir(".")
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Error reading directory: %v\n", err)
+		os.Exit(1)
+	}
+
+	for _, file := range files {
+		filePath := file.Name()
+		if file.IsDir() {
+			continue
+		}
```

Comment:
do we want to fsync recursively instead of continuing?

Reply:
also, does fsync not work on directory paths?

Reply:
oh yea oops, need to recurse. Fsync on directory path won't call fsync on the file. OFS people said that you need an explicit fsync on a file path to trigger flush to mount